app/views/users/show.html.erb

```diff
@@ -1,13 +1,28 @@
-<h2><%= @header %></h2>
+<div class="profile">
+  <div class="row">
+    <div class="header small-3 small-centered column">
+      <% if @user.photo %>
+        <div class="user-photo"> <%= image_tag(current_user.photo(:medium)) %> </div>
+      <% end %>
+      <h2><%= @user.full_name %></h2>
+    </div>
+  </div>
 
-<% if @user.photo %>
-  <%= image_tag @user.photo.url(:thumb) %>
-<% end %><br />
+  <div class="nav">
+    <h6>Your Information</h6>
+    <h6>Groups You Lead</h6>
```

Comment:
@shannonchu Remember to split up your features in different branches! We can merge this one in, but don't forget next time!